Source/FieldSolver/FiniteDifferenceSolver/FiniteDifferenceSolver.cpp

```diff
@@ -0,0 +1,65 @@
+/* Copyright 2020 Remi Lehe
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+
+#include "WarpXAlgorithmSelection.H"
+#ifdef WARPX_DIM_RZ
+#    include "FiniteDifferenceAlgorithms/CylindricalYeeAlgorithm.H"
+#else
+#    include "FiniteDifferenceAlgorithms/YeeAlgorithm.H"
+#    include "FiniteDifferenceAlgorithms/CKCAlgorithm.H"
+#    include "FiniteDifferenceAlgorithms/NodalAlgorithm.H"
+#endif
+#include "FiniteDifferenceSolver.H"
+#include "WarpX.H"
+
+/* \brief Initialize the finite-difference Maxwell solver (for a given refinement level)
```

Comment:
```suggestion
/** \brief Initialize the finite-difference Maxwell solver (for a given refinement level)
```
Also, this could in the `.H` file instead, at least for the `\brief` and `\param`, and the detailed description could stay there.